admin/system/delete_produk.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    function deleteDir($folder_path) {
+        if(!is_dir($folder_path)){
+            return false;
+        }
+
+        // List of name of files inside 
+        // specified folder 
+        $files = glob($folder_path.'/*');  
+        
+        // Deleting all the files in the list 
+        foreach($files as $file) { 
+        
+            if(is_file($file))  
```

Comment:
Codacy found an issue: [The use of function is_file() is discouraged](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)